### PR TITLE
Adapt for desktop

### PR DESCRIPTION
**Pull request by ivan-kolesen**

### CODE REVIEW

File: src/style/app.css [hunk 1]
```diff
@@ -479,3 +479,15 @@ body {
 .step-three-question {
   padding: 10vh;
   color: #000000; }
+
+@media (min-width: 960px) {
+  body:nth-child(2) {
```
Comment (davojta) [inline src/style/app.css]: very strange selector. Could we add comment for what we have it ?

Comment (ivan-kolesen) [reply src/style/app.css]: that is selector for 'root'. i've decided not to add className for the root-element and not to use selector '#root' in scss-file

Comment (davojta) [reply src/style/app.css]: It is ALWAYS better to use separate selector to style something - it is better in performance and better in support / reading the code )

Comment (davojta) [reply src/style/app.css]: @ivan-kolesen please add separate class to root

**Pull request merged**